docs/pages/application-access/reference/configuration.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Application Access Reference Documentation
+description: Configuration reference documentation for Teleport Application Access.
+---
+
+## Configuration
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+The following snippet shows the full YAML configuration of an Application Service
```

Comment:
Just a nit, this shows the section for `app_service` but on its own

Reply:
Good point - a follow-up ticket to this will make more significant improvements to the YAML.